proofs/isabelle/timing_trace.py

```
@@ -0,0 +1,169 @@
+        return None
+    finally:
+        con.close()
+    return row[0] if row and row[0] is not None else None
+
+
+def decompress(blob):
+    with tempfile.NamedTemporaryFile(delete=False, suffix=".blob") as f:
+        f.write(blob)
+        p = f.name
+    try:
+        r = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)